test/images/webhook/main.go

```diff
@@ -120,7 +122,34 @@ func admitConfigMaps(data []byte) *v1alpha1.AdmissionReviewStatus {
 	return &reviewStatus
 }
 
-type admitFunc func(data []byte) *v1alpha1.AdmissionReviewStatus
+func admitCRD(ar v1alpha1.AdmissionReview) *v1alpha1.AdmissionReviewStatus {
+	glog.V(2).Info("admitting crd")
+	cr := struct {
+		metav1.ObjectMeta
+		Data map[string]string
+	}{}
+
+	raw := ar.Spec.Object.Raw
+	err := json.Unmarshal(raw, &cr)
```

Comment:
about my Unstructured test question: this is e2e and makes sure that Unstructured arrives in the correct format :+1: